alpine/2.3.6/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Only alpine version that suports rethinkdb is edge
+FROM alpine:edge
+
+ENV RETHINKDB_VERSION 2.3.6-r1
```

Comment:
For consistency and compatibility with the images for other distros, this should be `RETHINKDB_PACKAGE_VERSION`, as it is in the Debian dockerfiles (see https://github.com/rethinkdb/rethinkdb-dockerfiles/blob/master/jessie/2.3.6/Dockerfile#L10 for an example).

Reply:
Any reason why you want to pin a version? Current release on Alpine 3.7 is -r4